packages/scorchwrap/src/buildtime/wrapper.js

```diff
@@ -0,0 +1,95 @@
+const { applySourceTransforms } = require("lavamoat-core");
+const diag = require("./diagnostics");
+const fs = require("fs");
+const q = JSON.stringify;
```

Comment:
change name?

Reply:
convention stolen from endo - it's a quick quoting function that doesn't fail for objects.